src/SFA.DAS.ApplyService.Application/Users/CreateNewContact/CreateNewContactRequest.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using MediatR;
+
+namespace SFA.DAS.ApplyService.Application.Users.CreateNewContact
+{
+    public class CreateNewContactRequest : IRequest
+    {
+        public Guid Id { get; set; }
+        public Guid OrganisationId { get; set; }
+        public string Email { get; set; }
+        public string GivenNames { get; set; }
+        public string FamilyName { get; set; }
+        public string DisplayName { get; set; }
```

Comment:
Is this used?

Reply:
Yep

Reply:
Whereabouts? I can't see it being used in the Handler